src/mlabns/mlabns.cpp

```diff
@@ -2,24 +2,12 @@
 // Measurement-kit is free software. See AUTHORS and LICENSE for more
 // information on the copying conditions.
 
-#include "src/ext/json/src/json.hpp"
-#include <functional>
-#include <initializer_list>
-#include <map>
-#include <measurement_kit/common.hpp>
-#include <measurement_kit/http.hpp>
-#include <measurement_kit/mlabns.hpp>
-#include <regex>
-#include <string>
-#include <utility>
-#include <vector>
-
-using json = nlohmann::json;
+#include "src/mlabns/mlabns.hpp"
 
 namespace mk {
 namespace mlabns {
 
-static ErrorOr<std::string> as_query(Settings &settings) {
+ErrorOr<std::string> as_query(Settings &settings) {
```

Comment:
I think that the whole content of this file should become the private header and that the `.cpp` file should not do much more than including the private header. You should see how these things are sketched out in NDT, where typically the private header with templates is named `$name_impl.hpp`.

Hence, I'd move the above function into the header, as a `static inline` function, under the argument that to ease code review it's better to have all relevant code into the same place (as opposed to having part in `.cpp` and part into the internal header file).